geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/JdbcPooledDataSourceFactory.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.geode.connectors.jdbc;
+
+import java.util.Properties;
+
+import javax.sql.DataSource;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+import org.apache.geode.datasource.PooledDataSourceFactory;
+
+/**
+ * This class implements PooledDataSourceFactory for the JDBC Connector extension.
+ * It will be used by default for a jndi-binding of type "POOLED".
+ * For more information see "gfsh create jndi-binding".
+ */
+public class JdbcPooledDataSourceFactory implements PooledDataSourceFactory {
+
+  public JdbcPooledDataSourceFactory() {}
```

Comment:
You can just delete empty no-arg constructors like this. Looks weird after spotless mangles it like this.